application/package/fedora/README.md

```diff
@@ -0,0 +1,21 @@
+# What is this?
+A simple ubuntu package for MultiMC that wraps the contains a script that downloads and installs real MultiMC on ubuntu based systems.
+
+It contains a `.desktop` file, an icon, and a simple script that does the heavy lifting.
+
+# How to build this?
+You need to install the following:
+
+```
+sudo dnf install fedora-packager fedpkg fedrepo_req copr-cli
+```
+
+Copy the `rpmbuild` folder to `rpmbuild_0.6.7` and then run:
+```
```

Comment:
Should use scripting to auto grab the tarball
See https://github.com/MultiMC/MultiMC5/blob/53ad5cb436ea4bffb93c7d26a1be43690a21bf0d/application/package/ubuntu/multimc/opt/multimc/run.sh#L5..L19 for a reference